libs/designsystem/src/lib/components/modal/services/modal.helper.ts

```diff
@@ -52,8 +52,8 @@ export class ModalHelper {
     const ionModal = await this.ionicModalController.create({
       component: config.flavor === 'compact' ? ModalCompactWrapperComponent : ModalWrapperComponent,
       cssClass: [
-        'kirby-overlay',
-        'kirby-modal',
+        config.flavor !== 'modal' ? 'kirby-overlay' : null,
+        config.flavor !== 'modal' ? 'kirby-modal' : null,
```

Comment:
@henrikvoetmand another thing to think about: When using the card modal, it will also remain swipeable on desktop, and it will be difficult for us to change that behavior. 

We should think of a good way to handle how to open different types of modals based on the platform. And consider if that is our job or an option exposed by the modal.